src/main/scala/edu/ie3/powerFactory2psdm/converter/NodeConverter.scala

```diff
@@ -66,12 +78,14 @@ object NodeConverter {
   def convertNode(
       node: Node,
       subnetId: Int,
-      voltLvl: VoltageLevel
+      voltLvl: VoltageLevel,
+      shortNodeId2Uuid: Map[String, UUID]
```

Comment:
Why choose a different name for the map here than above?

Also, specify in ScalaDoc